cogs5e/models/sheet/player.py

```diff
@@ -159,12 +159,15 @@ def new(cls, character, name, minv=None, maxv=None, reset=None, display_type=Non
             except d20.RollSyntaxError:
                 raise InvalidArgument(f"{reset_by} (`resetby`) cannot be interpreted as a number or dice string.")
 
-        # set initial value
-        initial_value = max(0, min_value or 0)
-        if reset_to_value is not None:
-            initial_value = reset_to_value
-        elif max_value is not None:
-            initial_value = max_value
+        # set initial value if not already set
+        if initial_value is None:
+            initial_value = max(0, min_value or 0)
+            if reset_to_value is not None:
+                initial_value = reset_to_value
+            elif max_value is not None:
+                initial_value = max_value
+        else:
+            initial_value = character.evaluate_math(initial_value)
```

Comment:
Would this allow values outside the specified min and max as an initial value?

Reply:
Indeed it would, was somewhat under the assumption that a person creating the counter would create it with a value within their own ranges.

I assume it should be clamped to the minimum and maximum?

Reply:
How does
```py
if initial_value is None: ...
else:
    initial_value = character.evaluate_math(initial_value)
    if min_value is not None:
        initial_value = max(initial_value, min_value)
    if max_value is not None:
        initial_value = min(initial_value, max_value)
```
look?

Reply:
LGTM